redisdb/tests/conftest.py

```diff
@@ -37,6 +37,7 @@ def __call__(self):
                     master.lpush('test_key1', 'test_value1')
                     master.lpush('test_key2', 'test_value2')
                     master.lpush('test_key3', 'test_value3')
+                    master.xadd('test_key4', {'test_value4': 'test_value4'})
```

Comment:
Related to my other commant:

```suggestion
                    master.xadd('test_key4', {'test_value4a': 'test_value4a'})
                    master.xadd('test_key4', {'test_value4b': 'test_value4b'})
```